### ISSUE TITLE
URL Names

### ISSUE THREAD

**Issue by williamgp11** [YOURLS/YOURLS#1955]
Hi!

Is there anyway we can change the format of the URL it generates?

I mean "site.com/1" is pretty short, but it looks weird, is there a plugin and/or option to maybe make it generate something like "site.com/aNiQ" (just some random letters) rather than having a single alphanumerical link.

Thank you,
William.

**Comment by dgw** [comment]
See #1946 and the other tickets referenced therein.

tl;dr please search before opening an issue, and make sure to look at closed issues too.
